### PR TITLE
{bio}[gimkl/2017b] DALTON v2016.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
DALTON 2016.2 built with gimkl/2017b.

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 4 out of 5 (3 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/bc2f8e5b658b1b1c3502332ac6318cb9 for a full test report.

**Comment by migueldiascosta** [comment]
@akesandgren, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

File: easybuild/easyconfigs/d/DALTON/DALTON-2016.2-gimkl-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+https://www.daltonprogram.org/www/citation.html.
+
+This is a OpenMP + MPI hybrid build.
+"""
+
+toolchain = {'name': 'gimkl', 'version': '2017b'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+# Dalton is licensed software. Need to download by hand.
+sources = ['%(name)s%(version)s-Source.tar']
```
Comment (boegel) [inline easybuild/easyconfigs/d/DALTON/DALTON-2016.2-gimkl-2017b.eb]: @akesandgren Now open source? https://gitlab.com/dalton/dalton/tree/Dalton2016.2

Comment (akesandgren) [reply easybuild/easyconfigs/d/DALTON/DALTON-2016.2-gimkl-2017b.eb]: Apparently yes... will fix.

Comment (akesandgren) [reply easybuild/easyconfigs/d/DALTON/DALTON-2016.2-gimkl-2017b.eb]: Not that easy. They assume recursive git clone...

**Comment by boegel** [comment]
@akesandgren I think we can close this one, and proceed with #5808 (I'm working on getting that one merged)?

**Comment by akesandgren** [comment]
Close in favor of #5808

**Pull request closed without merging**